common/src/test/java/io/pravega/common/concurrent/FuturesTests.java

```diff
@@ -75,18 +75,18 @@ public void testExceptionListener() {
     public void testExceptionallyCompose() {
         // When applied to a CompletableFuture that completes normally.
         val successfulFuture = new CompletableFuture<Integer>();
-        val f1 = Futures.exceptionallyCompose(successfulFuture, ex -> CompletableFuture.completedFuture(2));
+        val f1 = Futures.<Integer>exceptionallyCompose(successfulFuture, ex -> CompletableFuture.completedFuture(2));
```

Comment:
These three could also eliminate the warning, by providing an actual type rather than `val` which is perhaps a more natural solution.